app/bundles/LeadBundle/Config/config.php

```diff
@@ -971,5 +971,13 @@
         'contact_export_dir'                                                                    => '%mautic.application_dir%/media/files/temp',
         'contact_export_batch_size'                                                             => 20000,
         'contact_allow_multiple_companies'                                                      => true,
+        'peak_interaction_timer_cache_timeout'                                                  => 43800,
```

Comment:
I suggest providing a unit of time in the comment and giving e.g. hours as the basic unit